client-v2/src/components/Clipboard.tsx

```diff
@@ -62,13 +90,43 @@ interface ClipboardProps {
   removeSupportingCollectionItem: (parentId: string, id: string) => void;
   isClipboardOpen: boolean;
   toggleClipboard: (open: boolean) => void;
+  handleFocus: () => void;
+  handleArticleFocus: (articleFragment: TArticleFragment) => void;
+  handleBlur: () => void;
   dispatch: Dispatch;
+  focusedArticle?: string;
+  isClipboardFocused: boolean;
 }
 
+// Styled component typings for ref seem to be broken so any refs
+// passed to styled components has to be any for now.
+type Ref = any;
+type TClipboardWrapper = any;
```

Comment:
Sorry, couldn't do better than this

Reply:
That should work with a bit of HTMLElement but it's not the end of the world and styled components may well be the issue here

Reply:
Yes, I couldn't make it work with HTMLElement and seems to be an ongoing issue with styled components.